### PR TITLE
Improved documentation for QM requisitions

### PR DESCRIPTION
**Pull request by Kubius**
[CLEAN]
## About the PR 

Expands, and slightly cleans up, documentation of the requisition contract datums I made a while ago. This should not change the function of requisitions - just adds a whole lot of comment, and a bit of reorganization.

## Why's this needed? 

Requisitions are a neat spot for community contribution; less patchy documentation should hopefully make the experience more approachable.

### CODE REVIEW

File: code/modules/economy/requisition/rc_aid.dm [hunk 1]
```diff
@@ -1,4 +1,11 @@
 ABSTRACT_TYPE(/datum/req_contract/aid)
+/**
+ * Aid contracts are one of the three standard contract types.
```
Comment (Azrun) [inline code/modules/economy/requisition/rc_aid.dm:3]: Referring to this as 1/3 doesn't add much value and requires more maintenance if an additional type is added in the future.

File: code/modules/economy/requisition/rc_civilian.dm [hunk 2]
```diff
@@ -1,5 +1,10 @@
 ABSTRACT_TYPE(/datum/req_contract/civilian)
-/datum/req_contract/civilian //masters of the three seashells
+/**
+ * Civilian contracts are one of the three standard contract types.
```
Comment (Azrun) [inline code/modules/economy/requisition/rc_civilian.dm:3]: See previous

File: code/modules/economy/requisition/rc_scientific.dm [hunk 3]
```diff
@@ -1,5 +1,10 @@
 ABSTRACT_TYPE(/datum/req_contract/scientific)
-/datum/req_contract/scientific //adam savage dense gas voice: it's scientific!
+/**
+ * Scientific contracts are one of the three standard contract types.
```
Comment (Azrun) [inline code/modules/economy/requisition/rc_scientific.dm:3]: Again

File: code/modules/economy/requisition/requisition_contracts.dm [hunk 4]
```diff
@@ -102,24 +94,53 @@ ABSTRACT_TYPE(/datum/rc_entry/stack)
...
+	///Reagents being looked for in the evaluation; can be a list of several, or just the one.
+	var/chemname = "water"
+
+	rc_eval(atom/eval_item)
+		. = ..()
+		if(rollcount >= count) return // standard skip-if-complete
+		if(eval_item.reagents)
+			var/C
+			if(islist(src.chemname))
+				for(var/chemplural in src.chemname)
```
Comment (Azrun) [inline code/modules/economy/requisition/requisition_contracts.dm:115]: Naming doesn't make sense, at this point it should be a singular id.

File: code/modules/economy/requisition/rc_aid.dm (hunk 1 above)
Comment (Kubius) [reply code/modules/economy/requisition/rc_aid.dm]: Rectified this and other instances.

File: code/modules/economy/requisition/requisition_contracts.dm (hunk 4 above)
Comment (Kubius) [reply code/modules/economy/requisition/requisition_contracts.dm]: Variable renamed to chem_ids.

**Review by Azrun — APPROVED** [review]

**Pull request merged**